lib/Idream/Effects/Idris.hs

```diff
@@ -77,7 +80,11 @@ idrisGetLibDir = send IdrisGetLibDir
 
 idrisCompile :: Member Idris r
              => ProjectName -> PackageName -> Eff r ()
-idrisCompile projName pkgName = send $ IdrisCompile projName pkgName
+idrisCompile projName pkgName = send $ IdrisCompileLike projName pkgName CompileAction
```

Comment:
I don't like this approach, splitting up each effect shows more clear intent about what we try to do.
Also compiling and repl are too different to group together under 1 concept.
Better if we use the helper function approach I suggested elsewhere.